features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/themes/wso2/templates/conditions/manage/template.jag

```diff
@@ -78,17 +78,11 @@
 				<p><%=i18n.localize("Blacklisted items are not defined")%></p>
 			</div>
     <%} %>
-    <div class="form-actions">
-        <input id="add-condition-btn" type="button" class="btn btn-primary" value="<%=i18n.localize("Add")%>">
-    </div>
 </div>
 
 <script>
     $(document).ready(function(){
-        $('#add-condition-btn').on('click',function(){
-            location.href = "<%= jagg.url('/site/pages/conditions-edit.jag')%>";
-                });
-        });
+
     function editCondition(conditionId){
```

Comment:
Why define a function inside a document ready event? Take this function outside.